logging/logging.go

```diff
@@ -334,6 +396,10 @@ func monitoredResource(parent string) *mrpb.MonitoredResource {
 	}
 }
 
+func regionFromZone(zone string) string {
+	return zone[:strings.LastIndex(zone, "-")]
```

Comment:
Is there ever a case where this may not work as expected? (Maybe Anthos/GKE on-prem?) Will it handle the error properly?

Reply:
I think only GCF and GCR use region in the `location` label. The other services use zone. This is based on matching stdout logs ([see here](https://docs.google.com/document/d/1yL0YXM5grqmqMuab-lKHvzP_MvnFfmK8ys24GDFIWwQ/edit#heading=h.qmdgbrm3rdg3
)). pls cmiiw though!